ReactNativeClient/lib/models/Folder.js

```diff
@@ -190,6 +190,29 @@ class Folder extends BaseItem {
 		return output;
 	}
 
+	static async sortLocale(folders, orderDir = 'ASC') {
+
+		orderDir = orderDir.toUpperCase();
+
+		const output = folders.slice();
+
+		const mod = orderDir === 'ASC' ? +1 : -1;
+
+		const collator = new Intl.Collator(undefined, {
+			numeric: true,
+			sensitivity: 'base',
+		});
+
+		//  By passing the numeric: true option, it will smartly recognize numbers.
+		//  sensitivity: 'base' makes sorting CASE-INSENSITIVE
+
+		output.sort(function(a, b) {
+			return collator.compare(a.title, b.title) * mod;
+		});
```

Comment:
Does it work if `title` is `undefined`? It shouldn't happen normally but I think it can due to bug somewhere.

Reply:
Well, in this case, you cannot enter an undefined value, because Joplin would not allow a notebook with no name, but if the anyone of the variables is undefined it would be displayed the last

![Screenshot (8)](https://user-images.githubusercontent.com/54576074/77384888-93d16e80-6dac-11ea-9cf0-ab6996b93199.png)